nengo_bones/templates/static.sh.template

```diff
@@ -25,7 +25,9 @@ shopt -s globstar
     #   $ d removes a trailing newline
     sed -i -e 's/# $/#/g' -e '/get_ipython()/d' -e '$ d' -- docs/examples/**/*.py
     exe pylint docs --rcfile=setup.cfg --disable=missing-docstring,trailing-whitespace,wrong-import-position,unnecessary-semicolon
-    rm docs/examples/**/*.py
+    for notebook_file in docs/examples/**/*.ipynb; do
+        rm "${notebook_file%.ipynb}.py";
```

Comment:
I think the semicolon at the end here isn't necessary?

Reply:
Removed